scp: add -P example (#5027)
tldr-pages/tldr: pages/common/scp.md

> Secure copy.
> Copy files between hosts using Secure Copy Protocol over SSH.
> More information: <https://man.openbsd.org/scp>.

- Copy a local file to a remote host:

`scp {{path/to/local_file}} {{remote_host}}:{{path/to/remote_file}}`

- Use a specific port when connecting to the remote host:

> `scp {{path/to/local_file}} {{remote_host}}:{{path/to/remote_file}} -P {{port}}`

Reviewer: For me, I often use this format.

```suggestion
`scp {{path/to/local_file}} -P {{port}} {{remote_host}}:{{path/to/remote_file}}`
```

`scp {{path/to/local_file}} {{remote_host}}:{{path/to/remote_file}}`

> - Specify which port to connect to the remote host on:

Reviewer 2: Maybe this is better:
```suggestion
- Use a specific port when connecting to the remote host:
```
